doc/README.morello

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
We can drop the extraneous blank lines at the top.

Reply:
fixed in 1e441259371e1d238b0000424a4bc025b4408c2f